README.md

```diff
@@ -10,24 +10,24 @@ This is a collection of templates where much of this automation is pre-configure
 * Run Waf, which will build your entire project the first time you run it. Later, it will automatically figure out which parts of the project need to be rebuilt.
 
 
-Getting started 
+Getting started
 ----------------
 
-1. The template uses cookiecutter to enable personalized installations. Before you start, install cookiecutter on your system. 
+1. The template uses cookiecutter to enable personalized installations. Before you start, install cookiecutter on your system.
 
         $ pip install cookiecutter
 
-2. Make sure to have [Miniconda](http://conda.pydata.org/miniconda.html) or Anaconda installed. **Tested with Python 3.6 and higher.** Python 2 will not work. A modern LaTeX distribution (e.g. [TeXLive](www.tug.org/texlive/), [MacTex](http://tug.org/mactex/), or [MikTex](http://miktex.org/)) needs to be found on your path. 
+2. Make sure to have [Miniconda](http://conda.pydata.org/miniconda.html) or Anaconda installed. **Tested with Python 3.6 and higher.** Python 2 will not work. A modern LaTeX distribution (e.g. [TeXLive](www.tug.org/texlive/), [MacTex](http://tug.org/mactex/), or [MikTex](http://miktex.org/)) needs to be found on your path.
```

Comment:
Add that ``conda --version`` should show at least 4.6.12, else do ``conda update conda`` first.

Reply:
I have conda version 4.6.8 and I don't want to update right now, because according to some issues posted at conda, this may break my Powershell Use again. Do you mind waiting for an update of that until I have fully resolved windows conda environment use?

Reply:
No worries. Just make 4.6.x out of it.